pandunia-lekse.csv

```diff
@@ -40,9 +40,9 @@ airia|dex|IE|gle:Éire, eng:Ireland|Ireland||||||||||||||||Irlanti|
 airlaskeria||||air force||||||||||||||||ilmavoimat|
 airporte||||airport (aeroport)|aéroport|aeropuerto|aeroporto|аэропорт||飞机场|空港|공항|sân bay|विमानतल|বিমানবন্দর|bandar udara||havalimanı|flughaveno|lentoasema (lentokenttä)|
 ais|||eng:ice, deu:Eis, may:és|ice||||||冰|||||||||glacio|jää|lód
-aisehoke||||ice hockey|hockey sur glace|hockey sobre hielo|hóquei no gelo|хокке́й с ша́йбой||冰球| アイスホッケー|아이스하키||||||buz hokeyi||jääkiekko|hokej na lodzie
-aisekrem|||eng:ice cream, jpn:アイスクリーム (aisukurīmu), kor:아이스크림 (aiseukeurim), hin:आइसक्रीम (āisakrīm), urd:(āisakrīm), ben:আইসক্রীম (aisôkrīm), tha:ไอศกรีม (ai-sà-griim), may:es krim, swa:aiskrimu, tam:ஐஸ் கிரீம் (ais kirīm), tel:(ais krīm)|ice cream|crème glacée|helado|sorvete (gelado)|мороженое||冰淇淋|アイスクリーム|아이스크림||आइसक्रीम|আইসক্রীম|es krim|||glaciaĵo|jäätelö|lody
 aisi||||icy|||||||||||||||glacia|jäinen|lodowy
+aiskrem|||eng:ice cream, jpn:アイスクリーム (aisukurīmu), kor:아이스크림 (aiseukeurim), hin:आइसक्रीम (āisakrīm), urd:(āisakrīm), ben:আইসক্রীম (aisôkrīm), tha:ไอศกรีม (ai-sà-griim), may:es krim, swa:aiskrimu, tam:ஐஸ் கிரீம் (ais kirīm), tel:(ais krīm)|ice cream|crème glacée|helado|sorvete (gelado)|мороженое||冰淇淋|アイスクリーム|아이스크림||आइसक्रीम|আইসক্রীম|es krim|||glaciaĵo|jäätelö|lody
```

Comment:
me no abla okeya yi mute, sabu me fino fikra mome xudu uza -o- sa "kombolekse". mono kombolekse, du tena parte dul da mulu sa vokal, no nida ligovokal -o-.

me fata "cherry-pick" pa ali mute sa yi mutogrupe.

Reply:
a, me aha. we bon; ["too difficult to pronounce"](http://www.pandunia.info/engli/kanun.html#11._Word_building) no klar. sa sabu we, xudu muta kanun.html.